docs/shared/concepts/sync-generators.md

```diff
@@ -0,0 +1,46 @@
+# Sync Generators
+
+In Nx 19.7, you can use sync generators to keep applicable files in sync with your project graph. These can be for global configuration files or scripts, or at the task level to ensure to that project level configuration files are in sync before a task is run.
```

Comment:
I'm unsure if we're targeting 19.7 as the version for the stable release. There are still some PRs that need to land before that. @FrozenPandaz can confirm here.